Modules/_io/fileio.c

```diff
@@ -1293,8 +1293,8 @@ fileio_get_mode(PyObject *op, void *closure)
 static PyObject *
 fileio_get_blksize(PyObject *op, void *closure)
 {
-    fileio *self = PyFileIO_CAST(op);
 #ifdef HAVE_STRUCT_STAT_ST_BLKSIZE
+    fileio* self = PyFileIO_CAST(op);
```

Comment:
It looks like the `*` position accidentally became inconsistent in this file, which could be unfriendly with some editors.